libs/domains/services/feature/src/lib/service-state-chip/service-state-chip.tsx

```diff
@@ -16,10 +25,21 @@ function DeploymentStateChip({ environmentId, serviceId, ...props }: DeploymentS
   return <StatusChip status={deploymentStatus?.state ?? 'STOPPED'} {...props} />
 }
 
-type RunningStateChipProps = Omit<ServiceStateChipProps, 'mode'>
+interface RunningStateChipProps extends Omit<ServiceStateChipProps, 'mode'> {
+  withDeploymentFallback?: boolean
+}
 
-function RunningStateChip({ environmentId, serviceId, ...props }: RunningStateChipProps) {
+function RunningStateChip({
+  environmentId,
+  serviceId,
+  withDeploymentFallback = false,
+  ...props
+}: RunningStateChipProps) {
   const { data: runningStatus } = useRunningStatus({ environmentId, serviceId })
+  // NOTE: Some services don't have a live/running status like managed DB, we must fallback on the deployment status
```

Comment:
We have this case only for Managed DB not other services?

Reply:
Currently yes, I prefer to let parent component decide.
Otherwise I should have complex logic to determine which type of service it is.